test/benchee_test.exs

```diff
@@ -253,6 +253,40 @@ defmodule BencheeTest do
     end
   end
 
+  test ".run accepts atom keys for jobs" do
+    capture_io fn ->
```

Comment:
Well, we could still implement it as a tag so we could do `@tag :capture_io` in the same way they have `@tag :capture_log` now. Wouldn't be all that tricky, actually. It's totally not needed, but it might avoid others having the initial confusion that I had when I saw this. I thought there _must_ be some kind of typo or something because you were capturing the IO but not doing anything with it.